src/const.ts

```diff
@@ -233,6 +243,10 @@ export interface Constant extends Required<ConstantProps> {
    * @private Callback to be called when the widget open state changes.
    */
   onWidgetOpenStateChange?: (params: OnWidgetOpenStateChangeParams) => void;
+  /**
+   * @private Message input state controlling properties.
```

Comment:
Isn't this feature part of the public interface?

Reply:
ah I was wondering what was the meaning of this because I thought all Constant props are public. Ill change

Reply:
Mark it as public if you think a prop is okay to write in REAME and a real users are okay to use it. In this case, don't forget to add it to the prop table in README.
But a prop has been added just for an internal usage (i.e. bot studio), then mark it as private :)